kani-driver/src/cbmc_property_renderer.rs

```diff
@@ -421,6 +437,99 @@ pub fn format_result(
     result_str
 }
 
+pub fn format_result_coverage(properties: &Vec<Property>) -> String {
+    let mut coverage_checks: Vec<CoverageCheck> = Vec::new();
+
+    for prop in properties {
+        let status = &prop.status;
+        let description = &prop.description;
+        let location = &prop.source_location;
+
+        let mut check =
+            CoverageCheck::new("".to_string(), 0, description.to_string(), status.to_string());
```

Comment:
Not sure why you're creating a new type for coverage checks. You can just filter the properties with `property_class() == "coverage"`.